### DISCUSSION TITLE
Error after revert "Error initialising app: CouchDB error: missing"

### DISCUSSION THREAD

**Discussion by Kry9toN** [Budibase/budibase#14972] [Help]
hei someone can help me i cant to edit my app with this error

![image](https://github.com/user-attachments/assets/b4e650a7-8eea-4c3c-ac15-4c49fb6ff19d)

**Comment by ConorWebb96** [comment]
I've commented [here](https://github.com/Budibase/budibase/discussions/14631#discussioncomment-11165103).
